RetailCoder.VBE/UI/RubberduckUI.resx

```diff
@@ -1400,8 +1400,8 @@ All our stargazers, likers &amp; followers, for the warm fuzzies
   <data name="ParserState_ResolverError" xml:space="preserve">
     <value>Resolver Error</value>
   </data>
-  <data name="ParserState_Resolving" xml:space="preserve">
```

Comment:
@Hosch250 I mean the *key*, not its value. If anything the value can be whatever translation of "resolving" that's already there, it doesn't hurt. But a missing key means a missing caption when running in another language, and *that* is a problem.